bin/src/run/mergeOptions.js

```diff
@@ -12,6 +12,7 @@ export default function mergeOptions ( config, command ) {
 	const inputOptions = {
 		input: getOption('input'),
 		legacy: getOption('legacy'),
+		frozen: getOption('frozen'),
```

Comment:
As freezing is something that only influences the form of the output, I would consider this to be an output option only and not something that belongs to the general options as well, what do you think? Having this duplicity between input and output options seems to me rather confusing.

Reply:
I agree, good suggestion.